packages/gatsby-plugin-netlify-cms/src/gatsby-node.js

```diff
@@ -32,6 +32,20 @@ function deepMap(obj, fn) {
   return obj
 }
 
+exports.onCreateDevServer = ({ app, store }) => {
+  const { program } = store.getState()
+  app.get(`/admin`, function(req, res) {
+    res.sendFile(
+      path.join(program.directory, `public/admin/index.html`),
+      err => {
+        if (err) {
+          res.status(500).end()
```

Comment:
perhaps expose the err here?
```suggestion
          res.status(500).end(err.message)
```

we could also do check if the file exists before using sendFile and warn or error the user in console.